ledger/src/blockstore.rs

```diff
@@ -974,44 +972,46 @@ impl Blockstore {
         prev_inserted_shreds: &HashMap<ShredId, Shred>,
         leader_schedule_cache: &LeaderScheduleCache,
         reed_solomon_cache: &ReedSolomonCache,
-    ) -> Vec<Shred> {
-        let mut recovered_shreds = vec![];
+    ) -> Vec<Vec<Shred>> {
         // Recovery rules:
         // 1. Only try recovery around indexes for which new data or coding shreds are received
         // 2. For new data shreds, check if an erasure set exists. If not, don't try recovery
         // 3. Before trying recovery, check if enough number of shreds have been received
         // 3a. Enough number of shreds = (#data + #coding shreds) > erasure.num_data
-        for (erasure_set, working_erasure_meta) in erasure_metas.iter() {
```

Comment:
Might be easier to review by hiding the whitespace changes:

![hide-whitespaces](https://github.com/user-attachments/assets/b7ba3611-541b-4ec1-960b-c470672c8c45)